src/Sulu/Bundle/MediaBundle/Media/ImageConverter/FileImageLoader.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\ImageConverter;
+
+use Imagine\Exception\RuntimeException;
+use Imagine\Image\ImagineInterface;
+use Sulu\Bundle\MediaBundle\Media\Exception\GhostScriptNotFoundException;
+use Sulu\Bundle\MediaBundle\Media\Exception\ImageProxyMediaNotFoundException;
+use Sulu\Bundle\MediaBundle\Media\Exception\InvalidMimeTypeForPreviewException;
+use Sulu\Bundle\MediaBundle\Media\Exception\OriginalFileNotFoundException;
+use Sulu\Bundle\MediaBundle\Media\Video\VideoThumbnailServiceInterface;
+
+/**
+ * Loads the image from a media at the path located on the locale filesystem.
+ */
+class FileImageLoader implements ImageLoaderInterface
```

Comment:
Looks like this is not that easy possible... We can't only pass the content in this function, because ffmpeg and ghostscript need the path of the file, because under the hood they are calling them directly via `exec`. I might get rid of it for ghostscript, but not for ffmpeg...